CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased changes
+- Fix for https://github.com/Concordium/concordium-node/issues/147 so the correct state hash is now returned.
```

Comment:
We don't generally link issues like this. It's better to describe the bug directly.